src/primitives/polyline/styled.rs

```diff
@@ -246,6 +299,54 @@ mod tests {
         );
     }
 
+    #[test]
+    fn thick_stroke_translated() {
+        let mut display: MockDisplay<BinaryColor> = MockDisplay::new();
+
+        let styled = Polyline::new(&PATTERN)
+            .translate(Point::new(-4, -3))
+            .into_styled(PrimitiveStyle::with_stroke(BinaryColor::On, 4));
+
+        assert_eq!(
+            styled.bounding_box(),
+            Rectangle::new(Point::zero(), Size::new(28, 10))
+        );
+
+        styled.draw(&mut display).unwrap();
+
+        assert_eq!(
```

Comment:
Should we start to migrate to `display.assert_pattern(&[...])`?

Reply:
I can do! You mentioned you wanted to change it all in another PR so I decided to leave these as is to help with conflicts. I will change only the new tests in this PR to use it though - that should be relatively conflict free.

Reply:
I haven't started migrating the tests yet, so there is no danger of causing merge conflicts. But I did only meant to use it in the new tests anyway.